edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.MqttBrokerAdapter/handlers/SubscriptionChangeHandler.cs

```diff
@@ -82,16 +82,16 @@ public async Task<bool> HandleAsync(MqttPublishInfo publishInfo)
                                 ? this.identityProvider.Create(id1.Value, id2.Value)
                                 : this.identityProvider.Create(id1.Value);
 
-            var proxy = default(IDeviceListener);
-            var proxyMaybe = await this.connectionRegistry.GetDeviceListenerAsync(identity);
+            var listener = default(IDeviceListener);
+            var maybeListener = await this.connectionRegistry.GetDeviceListenerAsync(identity);
 
-            if (!proxyMaybe.HasValue)
+            if (!maybeListener.HasValue)
```

Comment:
Use GetOrElse

Reply:
How would that work if I want to return?